x-pack/plugins/reporting/server/usage/types.ts

```diff
@@ -109,44 +113,6 @@ export type ReportingUsageType = RangeStats & {
 export type ExportType = 'csv' | 'csv_searchsource' | 'printable_pdf' | 'PNG';
 export type FeatureAvailabilityMap = { [F in ExportType]: boolean };
 
-export interface KeyCountBucket {
-  key: string;
-  doc_count: number;
-}
-
-export interface AggregationBuckets {
-  buckets: KeyCountBucket[];
-}
-
-export interface StatusByAppBucket {
-  key: string;
-  doc_count: number;
-  jobTypes: {
-    buckets: Array<{
-      doc_count: number;
-      key: string;
-      appNames: AggregationBuckets;
-    }>;
-  };
-}
-
-export interface AggregationResultBuckets {
-  jobTypes: AggregationBuckets;
-  layoutTypes: {
-    doc_count: number;
-    pdf: AggregationBuckets;
-  };
-  objectTypes: {
-    doc_count: number;
-    pdf: AggregationBuckets;
-  };
-  statusTypes: AggregationBuckets;
-  statusByApp: {
-    buckets: StatusByAppBucket[];
-  };
-  doc_count: number;
-}
-
```

Comment:
These lines of code were mysteriously duplicated in the file

Reply:
🧙🏻‍♂️